programs/voter-stake-registry/src/instructions/close_voter.rs

```diff
@@ -62,12 +62,12 @@ pub fn close_voter<'key, 'accounts, 'remaining, 'info>(
     {
         let voter = ctx.accounts.voter.load()?;
 
-        let active_deposit_entries = voter.deposits.iter().filter(|d| d.is_used).count();
-        require_eq!(ctx.remaining_accounts.len(), active_deposit_entries);
-
         let any_locked = voter.deposits.iter().any(|d| d.amount_locked(curr_ts) > 0);
         require!(!any_locked, VsrError::DepositStillLocked);
 
+        let active_deposit_entries = voter.deposits.iter().filter(|d| d.is_used).count();
```

Comment:
Could you let me know why you moved that code? It's not an issue though, just looks odd.